documentation/wiki/FeatureAvailability.md

```diff
@@ -0,0 +1,12 @@
+# What is Feature Availablity?
+Feature Availablity is an API that can tell you a specific feature's availability status. Feature is saved as a string and availability is an enum `FeatureStatus`: `Undefined`, `Available`, `NotAvailable`, `Preview`.
```

Comment:
nit: This could perhaps use a sentence explaining what exactly we mean by 'feature', because without qualifications the word is quite ambiguous. I would specify that these are features of the MSBuild engine.